packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -517,7 +555,11 @@ class SliverGeometry {
   }
 }
 
+/// A hit test entry used by [RenderSliver].
```

Comment:
talk about the coordinate system used here?

Otherwise this doc is barely above our bar of "can't be written by someone who has just the name of the class". :-)